Gemfile

```diff
@@ -35,6 +35,12 @@ gem 'pg', '0.21.0'
 gem 'rails_email_validator'
 gem 'rake', '~> 11'
 gem 'ransack'
+# Adding `redis` as a direct dependency to highlight the fact that `sidekiq` in
+# version `3.3.4` has a very loose lock for `redis`, which causes the bug
+# described at: https://github.com/antirez/redis/issues/4272
+# N.B.: it will possibly need to be removed as soon as we bump `sidekiq` to the
+# latest version
```

Comment:
Should something also be added to the repo Readme? maybe in the section, `Background jobs`

Reply:
@margOnline It's not a sort of known bug than needs to be mentioned, it's a dependency issue that has been explained thoroughly in the commit description and the comment here. 3 places to mention this would sound too extreme to me.